connect_telegram_ext/extension.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "Telegram notify",
   "description": "A very simple Telegram notificator",
```

Comment:
Telegram notifications provide a mechanism to get fast notifications on desired events that happens at CloudBlue Connect platform in the scope of a given account, such as fulfillment requests, listings and a long etc...